src/main/java/pl/javastart/exercise/streams/FilteringMachineTwo.java

```diff
@@ -20,9 +19,7 @@
 
     public List<User> convertPeopleToUsers(List<Person> people) {
         return people.stream()
-                .map((Person t) -> User.getName(t))
-                .map(User::getAge)
-                .map(User::getNickname)
+                .map(person -> new User(person.getName(), person.getAge(), person.getName()+"_"+person.getAge()))
```

Comment:
jeszcze małe formatowanie tutaj bo naokoło + powinieneś dać spacje, ale to już możesz zrobić później :)